BMM.Api/Implementation/Models/Track.cs

```diff
@@ -52,16 +52,44 @@ public Track()
         public string InternalTitle { get; set; }
 
         [JsonIgnore]
-        public string GetUniqueKey => Id + "_" + Language;
+        public string GetUniqueKey => $"{Id}_{Language}";
+
+        public long LastPosition { get; set; }
```

Comment:
Nullable long?

Reply:
Default value for long is 0, so it fits the needs, because if LastPosition is not set we start from 0, so I would leave it as it is.